tests/client-dbus/.travis.yml

```diff
@@ -0,0 +1,18 @@
+os: linux
```

Comment:
You'll have to merge the important stuff in this .travis.yml into the one at the top-level, I think.

Reply:
It is just the lint target that gets invoked on Travis, right?  

I think I've got it merged.

Reply:
Exactly.

Reply:
ok